ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml

```diff
@@ -1855,6 +1855,36 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-agent-install-network-static
+- as: gcp-ipi-ovn-ipsec-f14
```

Comment:
I suggest to copy the same install workflow from amd nightly yaml file, and have some necessary setting for them.